pandas/_config/config.py

```diff
@@ -158,6 +158,9 @@ def _describe_option(pat: str = "", _print_desc: bool = True):
     s = ""
     for k in keys:  # filter by pat
         s += _build_option_description(k)
+        s += "\n"
+    # strip the last newline
+    s = s[: s.rfind("\n")]
```

Comment:
Is this last newline a problem?

Reply:
Solving the issue as you mentioned by joining the description.